.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-	"editor.formatOnSave": false,
+	"editor.formatOnSave": true,
```

Comment:
what is it for ? why you ve done it?

Reply:
for auto formatting purposes (see the group chat)